packages/sales-api-service/src/server/routes/staging-exceptions.js

```diff
@@ -1,8 +1,13 @@
 import { createStagingExceptionRequestSchema, createStagingExceptionResponseSchema } from '../../schema/staging-exception.schema.js'
-import { createTransactionFileException, createStagingException } from '../../services/exceptions/exceptions.service.js'
+import { createTransactionFileException, createStagingException, createDataValidationError } from '../../services/exceptions/exceptions.service.js'
 
 const SWAGGER_TAGS = ['api', 'staging-exceptions']
 
+const isDataValidationError = payload => {
+  const { statusCode } = JSON.parse(payload.transactionFileException.description)
```

Comment:
Wouldn't it be tidier to do `payload.code` here? https://hapi.dev/api/?v=20.1.4#-responsecodestatuscode